chat-plugins/help-ai.js

```diff
@@ -0,0 +1,43 @@
+
+'use strict';
+
+const OTHER_ADMIN_REQUESTS = 'http://www.smogon.com/forums/forums/other-admin-requests.346/';
+
+class HelpAI {
+	ask(question, command) {
+		question = question.toLowerCase(); // people yelling at the bot isn't very nice :(
+		if ((question.includes('forgot') || question.includes('reset')) && (question.includes('pw') || question.includes('pass'))) {
+			return `To get your password reset, PM a moderator (@) in the Help room - they can help walk you thru the process.  If no staff are online to help, you can submit a thread in the ${this.otherAdminRequests('Other Admin Requests Forum.')} Please be prepared to verify some questions regarding your account relating to when and where it was registered.`;
+		} else if ((question.includes('staff') || question.includes('mod') || question.includes('room owner')) && (question.includes('abusing') || question.includes('abuse'))) {
+			return `If you feel as though a <strong>room staff</strong> member has abused their authority, you can PM a Room Owner (#) from that room. Do <strong>/roomauth [roomname]</strong> to see if any are currently online (bolded on the list.) If it was a Room Owner that abused you, you can PM a Moderator (@) in the Help room with your claim and evidence.<br />` +
+				`If you feel as though a <strong>global staff</strong> member (moderator, driver) has abused their authority, you can submit a thread in the ${this.otherAdminRequests('Other Admin Requests Forum.')} Please be thorough in your report, explaining clearly what the problem is, how it happened, who it was with and providing as much proof as you currently have available of any wrongdoings. Be assured that ONLY Pokemon Showdown Senior Staff are able to view the post in that forum, and confidentially will be assured. Complaints are taken very seriously, and any false reports will be dealt with harshly.`;
```

Comment:
I don't think that this, the staff, or the pwreset one should be included here, as they're only really valid on main.